### DISCUSSION TITLE
How to rate a song?

### DISCUSSION THREAD

**Discussion by AndreKR** [navidrome/navidrome#2312] [Q&A] [answered]
I have `ND_ENABLESTARRATING` enabled but how do I actually rate a song? I don't see any stars.

**Comment by AndreKR** [comment, marked as answer]
Ah, nevermind, I have to search for the song in the Songs list. The stars aren't in the playlist view.

I think it would be a useful feature to be able to rate the currently playing song? But for now I can live with just favorite-ing songs.

**Reply by rodlcopter13** [reply]
There is already an issue for this topic.  
https://github.com/navidrome/navidrome/issues/2028
